extractor/server/server.go

```diff
@@ -5,32 +5,39 @@ import (
 	"net"
 	"net/http"
 	_ "net/http/pprof"
+	"strconv"
 
 	"github.com/src-d/berserker/extractor"
 
 	log "github.com/inconshreveable/log15"
 	"google.golang.org/grpc"
 )
 
-var profiler = flag.Bool("profiler", false, "start CPU & memeory profiler")
-var limit = flag.Uint64("limit", 0, "number of repositories, 0 = All from DB")
+var (
+	profiler         = flag.Bool("profiler", false, "start CPU & memeory profiler")
+	limit            = flag.Uint64("limit", 0, "number of repositories, 0 = All from DB")
+	maxGrpcMsgSizeMb = flag.String("max-message-size", "400", "maximum message size to send/receive to/from clients (in Mb)")
```

Comment:
I don't understand why you want to keep a number as a string. The validation below is totally valid using directly an int instead of a string as input.